frontend/src/hooks/useCollaborateData.tsx

```diff
@@ -39,6 +39,21 @@ interface IStaleNotificationOptions {
     afterSubmitAction: () => void;
 }
 
+const omitUnwantedData = (obj: any, field: string): any => {
```

Comment:
need to be properly typed since any looses all type information. 

chatgpt suggests sth like this :) 
```ts
type DeepOmit<T, K extends string> = {
  [P in keyof T]: P extends K ? never
                : T[P] extends object ? DeepOmit<T[P], K>
                : T[P]
};

function deepOmit<T, K extends string>(obj: T, key: K): DeepOmit<T, K> {
  if (typeof obj !== 'object' || obj === null) {
    return obj as any;
  }

  const output: any = Array.isArray(obj) ? [] : {};

  for (const k in obj) {
    if (k === key) continue;

    if (typeof obj[k] === 'object') {
      output[k] = deepOmit(obj[k], key);
    } else {
      output[k] = obj[k];
    }
  }

  return output;
}
```

Another option is to use a microlib doing that. The operation you're trying to do seems to be a deepOmit

Reply:
Also if you write your own deep omit it needs proper unit tests since it's a nontrivial function